### PR TITLE
Core: remove not needed logging

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

removes logging thats not needed anymore 

### Suggested Testing Steps

look at CI / build pcsx2

### CODE REVIEW

**Review by RedDevilus — APPROVED** [review]

**Comment by refractionpcsx2** [comment]
What about the other one?

File: pcsx2/HwRead.cpp [hunk 1]
```diff
@@ -291,7 +289,6 @@ static RETURNS_R64 _hwRead64(u32 mem)
 			// Current assumption: Reads 128 bits and discards the unused portion.
 
 			uint wordpart = (mem >> 3) & 0x1;
-			DevCon.WriteLn( Color_Cyan, "Reading 64-bit FIFO data (%s 64 bits discarded)", wordpart ? "upper" : "lower" );
```
Comment (lightningterror) [inline pcsx2/HwRead.cpp:292]: Can remove this line too.

File: pcsx2/HwRead.cpp [hunk 2]
```diff
@@ -69,8 +69,6 @@ mem32_t __fastcall _hwRead32(u32 mem)
 			
 			// Current assumption: Reads 128 bits and discards the unused portion.
```
Comment (lightningterror) [inline pcsx2/HwRead.cpp:71]: This line too.

File: pcsx2/HwRead.cpp [hunk 3]
```diff
@@ -66,10 +66,6 @@ mem32_t __fastcall _hwRead32(u32 mem)
 			// No game is known to attempt such a thing (yay!), so probably nothing for us to
 			// worry about.  Chances are, though, doing so is "legal" and yields some sort
 			// of reproducible behavior.  Candidate for real hardware testing.
-			
-			// Current assumption: Reads 128 bits and discards the unused portion.
```
Comment (lightningterror) [inline pcsx2/HwRead.cpp:70]: Comment should remain.

**Pull request merged**